src/commands/dev/lib/dev-router.ts

```diff
@@ -10,6 +10,25 @@ export default function(reqPath = '', routes?: RouteConfig[]): RouteResult {
   let found: RouteResult | undefined;
   const { pathname: reqPathname = '/' } = url.parse(reqPath);
 
+  const resolveRouteParameters = (str: string, dict: {keys: any, match: any}) => {
```

Comment:
```suggestion
  const resolveRouteParameters = (str: string, dict: {keys: string[], match: string[]}) => {
```

Looks like both are string arrays